lisa/features/__init__.py

```diff
@@ -64,6 +65,7 @@
     "PasswordExtension",
     "Resize",
     "ResizeAction",
+    "RunCommand",
```

Comment:
It's an Azure only feature, so put it in Azure, not shared place.

Reply:
I am still testing the performance of RunCommand vs SerialConsole for collecting logs.

RunCommand seems to be faster than Serial Console (including the time for reset password and login for serial console).

However, since RunCommand is only implemented in Azure platform, should we rather consider using Serial Console for the log collection?

Reply:
The ResetPassword feature relies on RunCommand or a similar underlying Azure capability. This means it only functions properly in Azure environments. If password authentication is disabled and access via the serial console is required, ResetPassword won’t work. However, if it offers improvements over RunCommand, we can consider using it.

Reply:
Remove the imports, the `RunCommand` is Azure only, it shouldn't be imported here.